---
title: "Predicting the severity of _C. difficile_ infections from the taxonomic composition of the gut microbiome"
author: Kelly L. Sovacool${^1}$, Sarah Tomkovich${^2}$, Megan L. Coden${^3}$, Patrick D. Schloss$^{2,4,\dagger}$
output:
    pdf_document:
        latex_engine: xelatex
        keep_tex: false
        includes:
          in_header: preamble.tex
    github_document:
        keep_html: false
fontsize: 11pt
geometry: margin=1.0in
---
```{r, include = FALSE}
schtools::set_knitr_opts()
knitr::opts_chunk$set(
  collapse = TRUE,
  comment = "#>",
  eval = TRUE,
  echo = FALSE
)
```
```{r data, include = FALSE}
library(tidyverse)
idsa_cases <- list(idsa_severe = 691L, 
                   not_severe = 468L)
ml_auroc = list(glmnet = 0.605812685055322,
                rf = 0.601858345021038,
                svmRadial = 0.577801153186847)
top5otu <- c(
    "_Clostridiales_ (OTU 195)",
    "_Lachnospiraceae_ (OTU 45)",
    "_Bacteroides_ (OTU 7)",
    "_Enterococcus_ (OTU 9)",
    "_Pasteurellaceae_ (OTU 30)"
)
```


\qquad $1$ Department of Computational Medicine and Bioinformatics, University of Michigan

\qquad $2$ Department of Microbiology and Immunology, University of Michigan

\qquad $3$ Department of Molecular, Cellular, and Developmental Biology, University of Michigan

\qquad $4$ Center for Computational Medicine and Bioinformatics, University of Michigan

\qquad $\dagger$ To whom correspondence should be addressed: pschloss@umich.edu

## Abstract

_C. difficile_ infection (CDI) can lead to adverse outcomes including recurrent 
infections, colectomy, and death. 
The composition of the gut microbiome plays an important role in 
determining colonization resistance and clearance when exposed to 
_C. difficile_.
We have 16S amplicon sequence data from CDI patient stool samples, with 
`r idsa_cases$idsa_severe` samples classified as severe CDI and 
`r idsa_cases$not_severe` as not severe according to the Infectious Diseases
Society of America (IDSA) definition.
IDSA defines severe CDI cases based on a white blood cell count \geq 15 k/{\mu}L
and serum creatinine level \geq 1.5 mg/dL.
Sequences were processed with mothur according to the MiSeq SOP and clustered
into _de novo_ OTUs at a 3% distance threshold.
We then trained machine learning (ML) models with OTU abundances as features to
predict the IDSA severity of CDI cases using the mikropml R package.
The dataset was randomly split into training and testing sets with 80% of the
data in the training set, then models were trained with 5-fold cross-validation
repeated 100 times, and performance as the area under the receiver-operator
curve (AUROC) was measured on the testing set for the best model.
This was repeated for 100 different random seeds and three different ML methods:
logistic regression, random forest, and support vector machines with a radial
basis kernel.
This process yielded median AUROC values of `r ml_auroc$glmnet` for logistic
regression, `r ml_auroc$rf` for random forest, and `r ml_auroc$svmRadial` for
support vector machines.
Feature importance was determined with a permutation test for the best random
forest model, revealing that the top 5 OTUs that contributed the most to model
performance were `r top5otu`.
The modest performance may be improved in future work by training to predict
clinically confirmed adverse patient outcomes rather than IDSA severity, such as
recurrence, admission to intensive care, colectomy, or death.
Predicting a patient's risk of experiencing a severe CDI and identifying the
specific microbiome features that distinguish severe CDI cases will allow
clinicians to tailor interventions based on each patient's individual
microbiome, ultimately leading to better health outcomes.

## Acknowledgements 

This research was supported by National Institutes of Health grants U01AI124255
and the Michigan Institute for Clinical and Health Research Postdoctoral
Translational Scholars Program (UL1TR002240 from the National Center for
Advancing Translational Sciences).
