---
title: "Predicting C. difficile infection severity from the taxonomic composition of the gut microbiome"
author:
    - name: Kelly L. Sovacool
      affil: 1
    - name: Sarah E. Tomkovich
      affil: 2
    - name: Megan L. Coden
      affil: 4
    - name: Vincent B. Young
      affil: "2,4"
    - name: Krishna Rao
      affil: 4
    - name: Patrick D. Schloss
      affil: "2,5"
affiliation:
    - num: 1
      address: Department of Computational Medicine & Bioinformatics, University of Michigan
    - num: 2
      address: Department of Microbiology & Immunology, University of Michigan
    - num: 3
      address: Department of Molecular, Cellular, and Developmental Biology, University of Michigan
    - num: 4
      address: Division of Infectious Diseases, Department of Internal Medicine, University of Michigan
    - num: 5
      address: Center for Computational Medicine and Bioinformatics, University of Michigan
poster_height: "42in"
primary_colour: "#00274C"
#secondary_colour: "#FFCB05"
#accent_colour: "#FFCB05"
logoright_name: "figures/triple-hex-logos.png"
logoleft_name: "figures/u-m_logo-hex.png"
output:
  posterdown::posterdown_html
bibliography: references.bib
csl: mbio.csl
---
```{r setup, include=FALSE}
knitr::opts_chunk$set(
  results = 'asis',
  echo = FALSE,
  comment = FALSE,
  warning = FALSE,
  message = FALSE,
  fig.align = 'center',
  fig.path = 'figures'
)
```

<style>
#main-img-right {
 height: 5%;
}
</style>

```{r deps}
library(here)
library(knitr)
library(tidyverse)
```

# Introduction

- _C. difficile_ infection (CDI) can lead to adverse outcomes including recurrent 
infections, colectomy, and death [@kwon_morbidity_2015]. 
- The composition of the gut microbiome plays an important role in 
determining colonization resistance and clearance when exposed to 
_C. difficile_ [@kociolek_breakthroughs_2016; @guh_clostridioides_2018].
- Identifying the specific microbiome features that distinguish severe CDI cases would eventually allow clinicians to tailor interventions based on a patient's risk, ultimately leading to better health outcomes.

**Dataset**

- We have 16S amplicon sequence data from `1191` CDI patient stool samples, with 
cases classified as severe or not severe according to three separate definitions:
  - the Infectious Diseases
Society of America (**IDSA**) definition: white blood cell count ≥ 15 k/μL
and serum creatinine level ≥ 1.5 mg/dL [@cohen_clinical_2010].
  - the CDC definition: ICU admission, colectomy, or death occurring within 30 days of CDI, and confirmed as **attributable** to CDI via clinical chart review.
  - **all-cause** severity: ICU admission, colectomy, or death occurring within 30 days of CDI, regardless of the cause.

```{r, message = FALSE}
read_csv('../../../data/process/outcomes_table.csv') %>% 
  kable()
```

# Methods

- Sequences were processed with mothur according to the MiSeq SOP and clustered
into _de novo_ OTUs at a 3% distance threshold [@kozich_development_2013; @schloss_introducing_2009].
- We then trained machine learning (ML) models with OTU abundances as features to
predict the IDSA severity, CDI-attributable severity, and all-cause severity of CDI cases using the mikropml R package [@topcuoglu_framework_2020; @topcuoglu_mikropml_2021].
- The dataset was randomly split into training and testing sets with 80% of the
data in the training set, then random forest models were trained with 5-fold cross-validation
repeated 100 times, and performance as the area under the receiver-operator
curve (AUROC) was measured on the testing set for the best model.
- This was repeated for 100 different random partitions of the data into training and testing sets.

```{r}
include_graphics('figures/mikropml-pipeline-logo.png')
```

# Results

- This process yielded median AUROC values of TODO
- Feature importance was determined with a permutation test for the best random forest model, revealing that the top 5 OTUs that contributed the most to model performance were TODO.

```{r}
include_graphics('../../../figures/plot_perf.png')
```

```{r}
include_graphics('../../../figures/plot_feat_imp.png')
```

# Conclusions 

- The modest performance may be improved in future work by training to predict
clinically confirmed adverse patient outcomes rather than IDSA severity, such as
recurrence, admission to intensive care, colectomy, or death.
- Predicting a patient's risk of experiencing a severe CDI and identifying the
specific microbiome features that distinguish severe CDI cases will allow
clinicians to tailor interventions based on each patient's individual
microbiome, ultimately leading to better health outcomes.

# Acknowledgements 

This research was supported by National Institutes of Health grants U01AI124255
and the Michigan Institute for Clinical and Health Research Postdoctoral
Translational Scholars Program (UL1TR002240 from the National Center for
Advancing Translational Sciences).

# References
