% Options for packages loaded elsewhere
\PassOptionsToPackage{unicode}{hyperref}
\PassOptionsToPackage{hyphens}{url}
%
\documentclass[
  11pt,
]{article}
\usepackage{lmodern}
\usepackage{amssymb,amsmath}
\usepackage{ifxetex,ifluatex}
\ifnum 0\ifxetex 1\fi\ifluatex 1\fi=0 % if pdftex
  \usepackage[T1]{fontenc}
  \usepackage[utf8]{inputenc}
  \usepackage{textcomp} % provide euro and other symbols
\else % if luatex or xetex
  \usepackage{unicode-math}
  \defaultfontfeatures{Scale=MatchLowercase}
  \defaultfontfeatures[\rmfamily]{Ligatures=TeX,Scale=1}
\fi
% Use upquote if available, for straight quotes in verbatim environments
\IfFileExists{upquote.sty}{\usepackage{upquote}}{}
\IfFileExists{microtype.sty}{% use microtype if available
  \usepackage[]{microtype}
  \UseMicrotypeSet[protrusion]{basicmath} % disable protrusion for tt fonts
}{}
\makeatletter
\@ifundefined{KOMAClassName}{% if non-KOMA class
  \IfFileExists{parskip.sty}{%
    \usepackage{parskip}
  }{% else
    \setlength{\parindent}{0pt}
    \setlength{\parskip}{6pt plus 2pt minus 1pt}}
}{% if KOMA class
  \KOMAoptions{parskip=half}}
\makeatother
\usepackage{xcolor}
\IfFileExists{xurl.sty}{\usepackage{xurl}}{} % add URL line breaks if available
\IfFileExists{bookmark.sty}{\usepackage{bookmark}}{\usepackage{hyperref}}
\hypersetup{
  hidelinks,
  pdfcreator={LaTeX via pandoc}}
\urlstyle{same} % disable monospaced font for URLs
\usepackage[margin=1.0in]{geometry}
\usepackage{graphicx}
\makeatletter
\def\maxwidth{\ifdim\Gin@nat@width>\linewidth\linewidth\else\Gin@nat@width\fi}
\def\maxheight{\ifdim\Gin@nat@height>\textheight\textheight\else\Gin@nat@height\fi}
\makeatother
% Scale images if necessary, so that they will not overflow the page
% margins by default, and it is still possible to overwrite the defaults
% using explicit options in \includegraphics[width, height, ...]{}
\setkeys{Gin}{width=\maxwidth,height=\maxheight,keepaspectratio}
% Set default figure placement to htbp
\makeatletter
\def\fps@figure{htbp}
\makeatother
\setlength{\emergencystretch}{3em} % prevent overfull lines
\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
\setcounter{secnumdepth}{-\maxdimen} % remove section numbering
\usepackage{fontspec}
\setmainfont{Times New Roman}
\usepackage{pdfpages}
\newlength{\cslhangindent}
\setlength{\cslhangindent}{1.5em}
\newenvironment{cslreferences}%
  {}%
  {\par}

\author{}
\date{\vspace{-2.5em}}

\begin{document}

\hypertarget{optifit-a-fast-method-for-fitting-amplicon-sequences-to-existing-otus}{%
\section{OptiFit: a fast method for fitting amplicon sequences to
existing
OTUs}\label{optifit-a-fast-method-for-fitting-amplicon-sequences-to-existing-otus}}

Kelly L. Sovacool\({^1}\), Sarah L. Westcott\({^2}\), M. Brodie
Mumphrey\({^1}\), Gabrielle Dotson\({^1}\), Patrick D. Schloss\({^2}\)

\(1\) Department of Computational Medicine and Bioinformatics,
University of Michigan, Ann Arbor, MI 48109

\(2\) Department of Microbiology and Immunology, University of Michigan,
Ann Arbor, MI 48109

\hypertarget{abstract}{%
\subsection{Abstract}\label{abstract}}

Assigning amplicon sequences to Operational Taxonomic Units (OTUs) is an
important step in characterizing the composition of microbial
communities across large datasets. OptiClust, a \emph{de novo} OTU
clustering method in the mothur program, has been shown to produce
higher quality OTU assignments than other methods and at comparable or
faster speeds (1, 2). A notable difference between \emph{de novo}
clustering and database-dependent methods is that OTU assignments
clustered with \emph{de novo} methods are not stable when new sequences
are added to a dataset (3). However, in some cases one may wish to
incorporate new samples into a previously clustered dataset without
performing clustering again on all sequences, such as when deploying a
machine learning model where OTUs are features (4). To provide an
efficient and robust method to fit amplicon sequence data to existing
OTUs, we developed the OptiFit algorithm as a new component of the
mothur program. To benchmark the OptiFit algorithm against \emph{de
novo} clustering with the OptiClust algorithm, we used four published
datasets isolated from soil (5), marine (6), mouse (7), and human (8)
samples. For each dataset, a subset of sequences was randomly selected
and clustered into OTUs with OptiClust, then the remaining sequences
were fit to the existing OTUs using the OptiFit algorithm. This was
repeated with subsets of varying sizes ranging from 10 to 90\% of
sequences in order to evaluate the bounds of the dataset size required
for OptiFit. Separately, all sequences were clustered with OptiClust to
provide a baseline of OTU assignment quality and runtime performance.
Each of these routines was repeated 10 times with different random seeds
to produce results that are robust to random variation. OTU quality was
evaluated using the Matthews Correlation Coefficient (MCC) with a
sequence similarity threshold of 97\% as described previously (3, 9). On
average, fitting sequences into existing OTUs with OptiFit performed 10
times faster than \emph{de novo} clustering with OptiClust, while the
average MCC scores produced were nearly indistinguishable across each
dataset. The OptiFit results across subset sizes ranging from 10 to 90\%
of sequences were also very similar, with slightly higher MCC scores for
larger subset sizes. Thus, OptiFit is an efficient way to fit new
sequences to existing OTUs yet without sacrificing the quality of OTU
assignments.

\hypertarget{references}{%
\subsection*{References}\label{references}}
\addcontentsline{toc}{subsection}{References}

\hypertarget{refs}{}
\begin{cslreferences}
\leavevmode\hypertarget{ref-westcott_opticlust_2017}{}%
1. \textbf{Westcott SL}, \textbf{Schloss PD}. 2017. OptiClust, an
Improved Method for Assigning Amplicon-Based Sequence Data to
Operational Taxonomic Units. mSphere \textbf{2}:e00073--17.
doi:\href{https://doi.org/10.1128/mSphereDirect.00073-17}{10.1128/mSphereDirect.00073-17}.

\leavevmode\hypertarget{ref-schloss_introducing_2009}{}%
2. \textbf{Schloss PD}, \textbf{Westcott SL}, \textbf{Ryabin T},
\textbf{Hall JR}, \textbf{Hartmann M}, \textbf{Hollister EB},
\textbf{Lesniewski RA}, \textbf{Oakley BB}, \textbf{Parks DH},
\textbf{Robinson CJ}, \textbf{Sahl JW}, \textbf{Stres B},
\textbf{Thallinger GG}, \textbf{Horn DJV}, \textbf{Weber CF}. 2009.
Introducing mothur: Open-Source, Platform-Independent,
Community-Supported Software for Describing and Comparing Microbial
Communities. Appl Environ Microbiol \textbf{75}:7537--7541.
doi:\href{https://doi.org/10.1128/AEM.01541-09}{10.1128/AEM.01541-09}.

\leavevmode\hypertarget{ref-westcott_novo_2015}{}%
3. \textbf{Westcott SL}, \textbf{Schloss PD}. 2015. De novo clustering
methods outperform reference-based methods for assigning 16S rRNA gene
sequences to operational taxonomic units. PeerJ \textbf{3}:e1487.
doi:\href{https://doi.org/10.7717/peerj.1487}{10.7717/peerj.1487}.

\leavevmode\hypertarget{ref-topcuoglu_effective_2019}{}%
4. \textbf{Topçuoğlu BD}, \textbf{Lesniak NA}, \textbf{Ruffin M},
\textbf{Wiens J}, \textbf{Schloss PD}. 2019. Effective application of
machine learning to microbiome-based classification problems. bioRxiv
816090. doi:\href{https://doi.org/10.1101/816090}{10.1101/816090}.

\leavevmode\hypertarget{ref-johnston_metagenomics_2016}{}%
5. \textbf{Johnston ER}, \textbf{Rodriguez-R LM}, \textbf{Luo C},
\textbf{Yuan MM}, \textbf{Wu L}, \textbf{He Z}, \textbf{Schuur EAG},
\textbf{Luo Y}, \textbf{Tiedje JM}, \textbf{Zhou J},
\textbf{Konstantinidis KT}. 2016. Metagenomics Reveals Pervasive
Bacterial Populations and Reduced Community Diversity across the Alaska
Tundra Ecosystem. Front Microbiol \textbf{7}.
doi:\href{https://doi.org/10.3389/fmicb.2016.00579}{10.3389/fmicb.2016.00579}.

\leavevmode\hypertarget{ref-henson_artificial_2016}{}%
6. \textbf{Henson MW}, \textbf{Pitre DM}, \textbf{Weckhorst JL},
\textbf{Lanclos VC}, \textbf{Webber AT}, \textbf{Thrash JC}. 2016.
Artificial Seawater Media Facilitate Cultivating Members of the
Microbial Majority from the Gulf of Mexico. mSphere \textbf{1}.
doi:\href{https://doi.org/10.1128/mSphere.00028-16}{10.1128/mSphere.00028-16}.

\leavevmode\hypertarget{ref-schloss_stabilization_2012}{}%
7. \textbf{Schloss PD}, \textbf{Schubert AM}, \textbf{Zackular JP},
\textbf{Iverson KD}, \textbf{Young VB}, \textbf{Petrosino JF}. 2012.
Stabilization of the murine gut microbiome following weaning. Gut
Microbes \textbf{3}:383--393.
doi:\href{https://doi.org/10.4161/gmic.21008}{10.4161/gmic.21008}.

\leavevmode\hypertarget{ref-baxter_microbiota-based_2016}{}%
8. \textbf{Baxter NT}, \textbf{Ruffin MT}, \textbf{Rogers MAM},
\textbf{Schloss PD}. 2016. Microbiota-based model improves the
sensitivity of fecal immunochemical test for detecting colonic lesions.
Genome Med \textbf{8}:37.
doi:\href{https://doi.org/10.1186/s13073-016-0290-3}{10.1186/s13073-016-0290-3}.

\leavevmode\hypertarget{ref-schloss_application_2016}{}%
9. \textbf{Schloss PD}. 2016. Application of a Database-Independent
Approach To Assess the Quality of Operational Taxonomic Unit Picking
Methods. mSystems \textbf{1}:e00027--16.
doi:\href{https://doi.org/10.1128/mSystems.00027-16}{10.1128/mSystems.00027-16}.
\end{cslreferences}

\end{document}
