---
output:
    pdf_document:
        latex_engine: xelatex
        keep_tex: true
        includes:
          in_header: preamble.tex
    github_document:
        keep_html: false
fontsize: 11pt
geometry: margin=1.0in
bibliography: ASM_Microbe_2020.bib
csl: mbio.csl
---

# OptiFit: a fast method for fitting amplicon sequences to existing OTUs

Kelly L. Sovacool${^1}$, Sarah L. Westcott${^2}$, M. Brodie Mumphrey${^1}$, Gabrielle A. Dotson${^1}$, Patrick D. Schloss${^2}$

$1$ Department of Computational Medicine and Bioinformatics, University of Michigan, Ann Arbor, MI 48109

$2$ Department of Microbiology and Immunology, University of Michigan, Ann Arbor, MI 48109

## Abstract

Assigning amplicon sequences to Operational Taxonomic Units (OTUs) is an important step in characterizing the composition of microbial communities across large datasets.
OptiClust, a _de novo_ OTU clustering method in the mothur program, has been shown to produce higher quality OTU assignments than other methods and at comparable or faster speeds [@westcott_opticlust_2017; @schloss_introducing_2009].
A notable difference between _de novo_ clustering and database-dependent methods is that OTU assignments clustered with _de novo_ methods are not stable when new sequences are added to a dataset [@westcott_novo_2015].
However, in some cases one may wish to incorporate new samples into a previously clustered dataset without performing clustering again on all sequences, such as when deploying a machine learning model where OTUs are features [@topcuoglu_effective_2019].
To provide an efficient and robust method to fit amplicon sequence data to existing OTUs, we developed the OptiFit algorithm as a new component of the mothur program.
To benchmark the OptiFit algorithm against _de novo_ clustering with the OptiClust algorithm, we used four published datasets isolated from soil [@johnston_metagenomics_2016], marine [@henson_artificial_2016], mouse [@schloss_stabilization_2012], and human [@baxter_microbiota-based_2016] samples.
For each dataset, a subset of sequences was randomly selected and clustered into OTUs with OptiClust, then the remaining sequences were fit to the existing OTUs using the OptiFit algorithm.
This was repeated with subsets of varying sizes ranging from 10 to 90% of sequences in order to evaluate the bounds of the dataset size required for OptiFit.
Separately, all sequences were clustered with OptiClust to provide a baseline of OTU assignment quality and runtime performance.
Each of these routines was repeated 10 times with different random seeds to produce results that are robust to random variation.
OTU quality was evaluated using the Matthews Correlation Coefficient (MCC) with a sequence similarity threshold of 97% as described previously [@schloss_application_2016; @westcott_novo_2015].
On average, fitting sequences into existing OTUs with OptiFit performed 10 times faster than _de novo_ clustering with OptiClust, while the average MCC scores produced were nearly indistinguishable across each dataset.
The OptiFit results across subset sizes ranging from 10 to 90% of sequences were also very similar, with slightly higher MCC scores for larger subset sizes.
Thus, OptiFit is an efficient way to fit new sequences to existing OTUs yet without sacrificing the quality of OTU assignments.

## References

